vllm/model_executor/models/llama.py

```diff
@@ -266,6 +267,7 @@ def forward(
         return hidden_states, residual
 
 
+@support_compile_llama_style
```

Comment:
Could these be part of the `ModelConfig` instead of requiring a modification to the model itself?

Reply:
it needs to be adjusted according to the model's signature. And multimodality models need to compile the vision tower and language part separately. I don't think it makes sense to add it in `ModelConfig` .

I would treat our `torch.compile` support similar as pipeline parallel, tensor parallel, quantization, etc, where every model needs to know how to adapt to these features.